python-spec/src/somacore/scene.py

```diff
@@ -36,17 +38,27 @@ class Scene(
     #     class Scene(  # type: ignore[type-var]
     #         ImplBaseCollection[ImplSOMAObject],
     #         somacore.Scene[
-    #             ImplCollection[Union[ImplDF, ImplNDArray]],  # _SpatialColl
-    #             ImplSOMAObject,                              # _RootSO
+    #             Union[ImplGeometryDataFrame, ImplPointCloud], # _SpatialDF
+    #             ImplImageCollection,                          # _ImageColl
+    #             ImplSOMAObject,                               # _RootSO
     #         ],
     #     ):
     #         ...
 
     __slots__ = ()
     soma_type: Final = "SOMAScene"  # type: ignore[misc]
 
-    obsl = _mixin.item[_SpatialColl]()
-    """A collection of spatial data defined on the obs data"""
+    img = _mixin.item[_ImageColl]()
+    """A collection of imagery backing the spatial data"""
 
-    varl = _mixin.item[collection.Collection[_SpatialColl]]()
-    """A collection of collections of spatial data defined on a measurement variable"""
+    obsl = _mixin.item[_SpatialDF]()
+    """A dataframe of the obs locations"""
+
+    varl = _mixin.item[collection.Collection[_SpatialDF]]()
+    """A collection of dataframes of the var locations"""
```

Comment:
@jp-dark - `varl` Collection - Is this indexed by `scene_id` and the value is a dataframe that has: `obs_id`, `var_id`, and some geometry columns?

Reply:
> varl Collection - Is this indexed by scene_id and the value is a dataframe that has: obs_id, var_id, and some geometry columns?

No, the collection is indexed by the measurement name. The value is a dataframe that has `var_id`, some geometry columns, and possibly other accessory data.